content/docs/guides/ads_on_amp.md

```diff
@@ -0,0 +1,17 @@
+---
+$title: Ads on AMP
+$order: 8
+---
+<div class="toc">
+{% include "/views/partials/sub_nav.html" %}
+  <ul>
+    <li><a href="/learn/who/#ads">Supported Platforms</a></li>
+    <li><a href="/roadmap">Roadmap</a></li>
+  </ul>
+</div>
+
+AMP pages are just web pages, except consistently faster. Many of the best practices in this guide are consistent across non-AMP pages.
```

Comment:
@pbakaus the point here is to lead with the benefit, not the features. i get that "just faster" is a drastic oversimplification, but can we frame the "custom web components and optimized delivery" factors as benefits rather than features?